volatility3/framework/plugins/windows/psxview.py

```diff
@@ -0,0 +1,279 @@
+import datetime, logging
+
+from volatility3.framework import constants, exceptions
+from volatility3.framework.interfaces import plugins
+from volatility3.framework.configuration import requirements
+from volatility3.framework.renderers import format_hints, TreeGrid
+from volatility3.plugins.windows import (
+    handles,
+    info,
+    pslist,
+    psscan,
+    sessions,
+    thrdscan,
+)
+
+vollog = logging.getLogger(__name__)
+
+
+class PsXView(plugins.PluginInterface):
+    """Lists all processes found via 6 of the methods described in \"The Art of Memory Forensics,\" which may help
+    identify processes that are trying to hide themselves. I recommend using -r pretty if you are looking at this
+    plugin's output in a terminal."""
+
+    # I've omitted the desktop thread scanning method because Volatility3 doesn't appear to have the funcitonality
+    # which the original plugin to do it.
+
+    # I don't think it's worth including the sessions method either because both the original psxview plugin
+    # and Volatility3's sessions plugin begin with the list of processes found by PsList.
+    # The original psxview plugin's session code essentially just filters the pslist for processes
+    # whose session ID is not None. I've matched this in my code, but again, it doesn't seem worth including.
+
+    # Lastly, I've omitted the pspcid method because I could not for the life of me get it to work. I saved the
+    # code I do have from it, and will happily share it if anyone else wants to add it.
+
+    _required_framework_version = (2, 0, 0)
+    _version = (1, 0, 0)
+
+    @classmethod
+    def get_requirements(cls):
+        return [
+            requirements.ModuleRequirement(
+                name="kernel",
+                description="Windows kernel",
+                architectures=["Intel32", "Intel64"],
+            ),
+            requirements.VersionRequirement(
+                name="info", component=info.Info, version=(1, 0, 0)
+            ),
+            requirements.VersionRequirement(
+                name="pslist", component=pslist.PsList, version=(2, 0, 0)
+            ),
+            requirements.VersionRequirement(
+                name="psscan", component=psscan.PsScan, version=(1, 0, 0)
+            ),
+            requirements.VersionRequirement(
+                name="thrdscan", component=thrdscan.ThrdScan, version=(1, 0, 0)
+            ),
+            requirements.VersionRequirement(
+                name="handles", component=handles.Handles, version=(1, 0, 0)
+            ),
+            requirements.VersionRequirement(
+                name="sessions", component=sessions.Sessions, version=(0, 0, 0)
+            ),
+            requirements.BooleanRequirement(
+                name="identify-expected",
```

Comment:
Check that this does get stored in the config with a `-` rather than a `_`

Reply:
I ran the plugin with the command-line arguments with no issues.